lib/rules/web/admin_console/4.5/alert_manager.xyaml

```diff
@@ -0,0 +1,10 @@
+browse_to_receiver_config:
+  element:
+    selector:
+      xpath: //*[@role='gridcell' and text()='<receiver>']/following-sibling::td//*[text()='Configure']
+    op: click
+  action: wait_box_loaded
+set_pagerduty_key:
+  params:    
+    input_field_id: integration-key
+  action: set_input_value
```

Comment:
all these lines can be moved to `monitoring.xyaml` file, we don't need create a new file